images/ceph/Dockerfile

```diff
@@ -19,8 +19,12 @@ FROM BASEIMAGE
 ARG S5CMD_VERSION
 ARG S5CMD_ARCH
 
-# install 'ip' tool for Multus
-RUN dnf install -y --repo baseos --setopt=install_weak_deps=False iproute && dnf clean all
+# 'ip' tool must be installed for Multus. It's present in the base image today, but it will likely
+# be removed in the future. Doing a 'dnf install' sometimes breaks CI when centos repos go down or
+# have other package build errors. In order to make sure Rook CI catches the eventual removal and
+# also limit Rook CI breakage due to centos breakage, simply check that 'ip' is present.
+# Eventually: dnf install -y --repo baseos --setopt=install_weak_deps=False iproute && dnf clean all
+RUN which ip
```

Comment:
This fails the build if not found?

Reply:
Correct. In the eventual future, when `ip` is removed from the image, CI will start failing, and we will need to add the `dnf install` (or equivalent) line back to the build. But until then, this should succeed without adding a hard dependency on CentOS repos.